### PR TITLE
Fix spacing in kernel dependencies page

### PR DESCRIPTION
**Pull request by hjiawei**
This change fixes spacing in the kernel dependencies page and reformat README.md.

Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,13 +24,14 @@ Or
...
 ```
+
 or
 
 ```bash
 make start CONTAINERIZED=true
 ```
 
-This command starts a local development server on http://localhost:3000. Most changes are reflected live without
+This command starts a local development server on <http://localhost:3000>. Most changes are reflected live without
```
Comment (ctauchen) [inline README.md]: Let's remove this change. We don't need this URL to be linked.

**Review by ctauchen — COMMENTED** [review]
Thanks for the cleanup. One small adjustment, otherwise LGTM.

File: README.md (hunk 1 above)
Comment (hjiawei) [reply README.md]: This change is reverted.

**Comment by ctauchen** [comment]
LGTM, pending rebase to resolve merge conflict.

**Comment by hjiawei** [comment]
> LGTM, pending rebase to resolve merge conflict.

Rebased and conflicts are resolved.

**Pull request merged**